packages/react-i18n/src/test/i18n.test.ts

```diff
@@ -1361,6 +1361,53 @@ describe('I18n', () => {
           ).toBe('4:00 p.m.');
         });
 
+        it('formats a date less than one week away', () => {
+          const fiveDaysInMilliseconds = 5 * 24 * 60 * 60 * 1000;
+          const today = new Date('2012-12-20T00:00:00-00:00');
+          const lessThanOneWeekAway = new Date(
+            today.getTime() + fiveDaysInMilliseconds,
+          );
+          clock.mock(today);
+          const i18n = new I18n(defaultTranslations, {
+            ...defaultDetails,
+            timezone,
+          });
+
+          i18n.formatDate(lessThanOneWeekAway, {
+            style: DateStyle.Humanize,
+          });
+          expect(translate).toHaveBeenCalled();
```

Comment:
```suggestion
          expect(translate).toHaveBeenCalledWith(
            'date.humanize.lessThanOneWeekAway',
            {
              pseudotranslate: false,
              replacements: {weekday: 'Tuesday', time: '11:00 a.m.'},
            },
            defaultTranslations,
            i18n.locale,
          );
```

This suggestion makes the assumption that you're using the new key (`lessThanOneWeekAway`)